ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -112,6 +119,22 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
         super.ngOnDestroy();
         this.ngUnsubscribe.next();
         this.ngUnsubscribe.complete();
+        window.removeEventListener('scroll', this.onScroll, true);
+    }
+
+    @boundMethod
+    public onScroll(event): void {
+        const isAutoCompleteOptionsScrolling = event.target === this.autocompleteSource.panel?.nativeElement;
+
+        if (this.autocompleteTrigger.panelOpen && !isAutoCompleteOptionsScrolling) {
+            this.autocompleteTrigger.closePanel();
+        }
+    }
+
+    public openAutocompleteOptions(): void {
+        if (!this.autocompleteTrigger.panelOpen) {
+            this.autocompleteTrigger.openPanel();
+        }
     }
```

Comment:
Actually, forget what I said earlier. Just use the arrow function. Looks a little odd when reading the code, but it is preferable to adding another dependency that has not been updated since 2018. 
Sorry for the flip-flopping!

Reply:
Sure, will do that